README.md

```diff
@@ -565,6 +573,32 @@ shopify.metafield.create({
 where `params` is a plain JavaScript object. See https://help.shopify.com/api/reference?ref=microapps
 for parameters details.
 
+
+### GraphQL
+
+The `shopify` instance also allows to use the GraphQL API through the `graphql` method, which returns a
```

Comment:
Please keep the limit of 80 chars per line everywhere.

Reply:
Ok